### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by Sakurann**

### CODE REVIEW

**Comment by Sakurann** [comment]
Issue #90

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 1]
```diff
@@ -122,40 +125,40 @@ Section 2 of [@!RFC7515].
...
 
-Holder
-:  An entity that received SD-JWTs from the Issuer and has control over them.
+Holder:
+  An entity that received SD-JWTs from the Issuer and has control over them.
 
-Verifier
-:  An entity that requests, checks and extracts the claims from HS-Disclosures JWT.
+Verifier:
+  An entity that requests, checks and extracts the claims from HS-Disclosures JWT.
```
Comment (danielfett) [inline draft-ietf-oauth-selective-disclosure-jwt.md:161]: This is not the proper formatting. The formatting we're currently using is correct in terms of markdown (see https://www.markdownguide.org/extended-syntax/#definition-lists) and it gets rendered correctly into the XML, but xml2rfc does not seem to do a good job at rendering it. We should look into that.

**Review by danielfett — CHANGES_REQUESTED** [review]

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 1 above)
Comment (danielfett) [reply draft-ietf-oauth-selective-disclosure-jwt.md:161]: I checked again and it seems that others (e.g., RFC9000) also use definition lists, but add another colon after the term to be defined.

Comment (Sakurann) [reply draft-ietf-oauth-selective-disclosure-jwt.md:161]: > Verifier:
>:  An entity that requests, checks and extracts the claims from HS-Disclosures JWT.

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 2]
```diff
@@ -89,24 +89,27 @@ cases as well.
 
 This document also describes an optional mechanism for Holder Binding,
 or the concept of binding an SD-JWT to key material controlled by the
-Holder.
+Holder. Strenght of the Holder Binding is conditional upon the trusts
+in protection of the private key of the key matetial an SD-JWT is bound to.
```
Comment (danielfett) [inline draft-ietf-oauth-selective-disclosure-jwt.md:93]:
```suggestion
Holder. The strength of the Holder Binding is conditional upon the trust
in the protection of the private key of the key pair an SD-JWT is bound to.
```

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 3]
```diff
@@ -1301,7 +1305,11 @@ TBD
 
 All of the following examples are non-normative.
 
-## Example 2a - Structured SD-JWT
+## Structured SD-JWT
+
+The following examples show the variations of a structured SD-JWT.
```
Comment (danielfett) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1310]:
```suggestion
The following examples show variations of a structured SD-JWT.
```

**Review by danielfett — CHANGES_REQUESTED** [review]

**Comment by danielfett** [comment]
Also this doesn't compile due to trailing spaces.

**Review by danielfett — APPROVED** [review]

**Pull request merged**